src/VisualStudio/CSharp/Impl/ProjectSystemShim/EntryPointFinder.cs

```diff
@@ -19,7 +17,13 @@ public static IEnumerable<INamedTypeSymbol> FindEntryPoints(INamespaceSymbol sym
         {
             var visitor = new EntryPointFinder();
             // Only search source symbols
-            visitor.Visit(symbol.ContainingCompilation.SourceModule.GlobalNamespace);
+            // Some callers will give a symbol that is not part of a compilation
```

Comment:
can we fix those callers?  don't we own all callers?

Reply:
No, this is used by the project system and others that have internals-visible-to. I think the correct fix is to refactor this api to take a compilation instead of a symbol but that will take a long time to update all the partners who use this. I will submit a follow up PR for that after this but I want to fix this regression before it goes out to customers.

Reply:
see https://github.com/dotnet/roslyn/pull/60614 for the API refactoring and ttps://github.com/dotnet/project-system/pull/8044 for the API consumption